src/std/rpc/ws_client/mod.rs

```diff
@@ -139,13 +139,27 @@ where
     }
 }
 
+const MAX_MESSAGE_LEN: usize = 1024;
+
+fn format_message(msg: &Message) -> String {
+    match (msg.as_text(), msg) {
```

Comment:
The original implementation of the `Display` for the `Message` ignores the content of the binary and only prints something like "Binary Data<length=n>" in the case when as_text() fails. Let's do similarly here and only match on `msg.as_text()`.

Reply:
done

Reply:
one last nit, there is no need to match on `(msg.as_text(), msg)` simple match on `msg.as_text()` and then using `msg.len()` instead of `bin.len()` will do the work.

Reply:
oh, indeed, I must have missed that `Message::len`, thanks - done